src/parser/parser.yy

```diff
@@ -2769,15 +2768,6 @@ delete_vertex_sentence
     }
     ;
 
-download_sentence
```

Comment:
Maybe we could remain the syntax.

Reply:
I'm not sure. Remove `DOWNLOAD` and `INGEST` sentences that maybe could unify the execution semantics.

Reply:
You could still add the job syntax but keep these for compatible.